src/commands/start.js

```diff
@@ -55,8 +58,18 @@ async function start(opts: *) {
   const directory = process.cwd();
   const configPath = getWebpackConfigPath(directory, opts.config);
 
+  let assetsDest;
+  if (opts.assetsDest) {
+    assetsDest = path.isAbsolute(opts.assetsDest)
+      ? opts.assetsDest
+      : path.join(directory, opts.assetsDest);
+  } else {
+    assetsDest = path.join(os.tmpdir(), fs.mkdtempSync('haul-start-'));
```

Comment:
I'm afraid `fs.mkdtempSync` will override the cache every time you run `haul start`, making Haul always start with a cold cache. Or am I getting it wrong?
Also you can move this assignment to line 61 and remove the `else` branch

Reply:
> I'm afraid `fs.mkdtempSync` will override the cache every time you run `haul start`, making Haul always start with a cold cache. Or am I getting it wrong?

Currently `haul start` does not re-use previously generated bundles - it always rebuilds the bundle (code cache) upon first request. IMO this is correct behavior as there might be source changes between 2 runs of `haul start`.

> Also you can move this assignment to line 61 and remove the `else` branch

Moving line 67 out of `else` would create a temp folder regardless of `opts.assetsDest`. That might not be ideal..

Reply:
You're right.

> IMO this is correct behavior as there might be source changes between 2 runs of `haul start`.

It should use cache whenever possible, but let's keep current behavior for now.